src/css/_style.scss

```diff
@@ -301,7 +309,7 @@ button {
 }
 
 .hidden {
-  display: none;
+  display: none !important;
```

Comment:
We can put the .hidden declaration at the bottom of the css file and remove !important